QuizzApp/Core/Custom+UI/LogOutAlertViewController.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  LogOutAlert.swift
+//  QuizzApp
+//
+//  Created by bakhva  on 19.06.23.
+//
+
+import UIKit
+
+protocol logOutAlertDelegate: AnyObject {
+    func pressedYesOnLogOut()
+    func pressedNoOnLogOut()
+}
+
+final class LogOutAlertViewController: UIViewController {
+    
+    weak var delegate: logOutAlertDelegate?
+    
+    // MARK: Components
+    private let alertView: UIView = {
+        let view = UIView()
+        view.backgroundColor = QuizzAppColors.buttonColor
+        view.layer.cornerRadius = Constants.alertViewRadius
+        view.translatesAutoresizingMaskIntoConstraints = false
+        
+        return view
+    }()
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.font = .boldSystemFont(ofSize: Constants.titleLabelFont)
+        label.textColor = .systemBackground
+        label.textAlignment = .center
+        label.numberOfLines = Constants.titleLabelNumberOfLines
+        label.translatesAutoresizingMaskIntoConstraints = false
+        
+        return label
+    }()
+    
+    private lazy var yesButton = createButton(Constants.yesButtonTitle)
+    
+    private lazy var noButton = createButton(Constants.noButtonTitle)
+    
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView()
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.addArrangedSubview(noButton)
+        stackView.addArrangedSubview(yesButton)
+        stackView.axis = .horizontal
+        stackView.spacing = Constants.stackViewSpacing
+        
+        return stackView
+    }()
+    
+    // MARK: ViewDidLoad
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configUI()
+        addViews()
+        constraints()
+    }
+    
+    func setTitleText(_ text: String) {
+        titleLabel.text = text
+    }
+}
+
+// MARK: -Private Functions
+private extension LogOutAlertViewController {
+    // MARK: Config UI
+    func configUI() {
+        view.backgroundColor = Constants.backgroundViewColor
+        yesButton.addTarget(self, action: #selector(yesPressed), for: .touchUpInside)
+        noButton.addTarget(self, action: #selector(noPressed), for: .touchUpInside)
+    }
+    
+    // MARK: Create Button
+    func createButton(_ text: String) -> UIButton {
```

Comment:
ეფლის რჩევაა რომ გამოვიყენოთ სიტყვა `make` მსგავს შემთხვევებში.
```suggestion
    func makeButton(_ text: String) -> UIButton {
```